packages/desktop-client/src/components/accounts/Filters.js

```diff
@@ -244,7 +244,7 @@ function ConfigureField({
               field={field}
               subfield={subfield}
               type={type === 'id' && op === 'contains' ? 'string' : type}
-              value={normalizeValue(value, op === 'oneOf')}
```

Comment:
I reverted this PR basically https://github.com/actualbudget/actual/pull/855

The new implementation actually exposes a bunch of other issues that are not so simple to address. So I'll revert and re-open the issue. And hopefully get to fixing it sometime later.